app/src/main/java/com/hover/stax/requests/NewRequestFragment.kt

```diff
@@ -226,12 +226,12 @@ class NewRequestFragment : AbstractFormFragment(), PushNotificationTopicsInterfa
         val recipientError = requestViewModel.requesteeErrors()
         requesteeInput.setState(recipientError, if (recipientError == null) AbstractStatefulInput.SUCCESS else AbstractStatefulInput.ERROR)
 
-        requestViewModel.activeAccount.value?.let {
-            if (!requestViewModel.isValidAccount()) {
-                payWithDropdown.setState(getString(R.string.incomplete_account_setup_header), AbstractStatefulInput.ERROR)
-                return false
-            }
-        }
+//        requestViewModel.activeAccount.value?.let {
+//            if (!requestViewModel.isValidAccount()) {
+//                payWithDropdown.setState(getString(R.string.incomplete_account_setup_header), AbstractStatefulInput.ERROR)
+//                return false
+//            }
+//        }
```

Comment:
Could you please add a note on why it is being commented out?

Reply:
because an account doesn't have to be valid to request money to it